phpunit/block-supports/typography-test.php

```diff
@@ -316,291 +316,603 @@ public function test_gutenberg_get_typography_font_size_value( $font_size, $shou
 	public function data_generate_font_size_preset_fixtures() {
 		return array(
 			'returns value when fluid typography is deactivated' => array(
-				'font_size'                   => array(
+				'font_size'       => array(
 					'size' => '28px',
 				),
-				'should_use_fluid_typography' => false,
-				'expected_output'             => '28px',
+				'settings'        => null,
+				'expected_output' => '28px',
 			),
 
 			'returns value where font size is 0'         => array(
-				'font_size'                   => array(
+				'font_size'       => array(
 					'size' => 0,
 				),
-				'should_use_fluid_typography' => true,
-				'expected_output'             => 0,
+				'settings'        => array(
+					'typography' => array(
+						'fluid' => true,
+					),
+				),
+				'expected_output' => 0,
 			),
 
 			"returns value where font size is '0'"       => array(
-				'font_size'                   => array(
+				'font_size'       => array(
 					'size' => '0',
 				),
-				'should_use_fluid_typography' => true,
-				'expected_output'             => '0',
+				'settings'        => array(
+					'typography' => array(
+						'fluid' => true,
+					),
+				),
+				'expected_output' => '0',
 			),
 
 			'returns value where `size` is `null`'       => array(
-				'font_size'                   => array(
+				'font_size'       => array(
 					'size' => null,
 				),
-				'should_use_fluid_typography' => false,
-				'expected_output'             => null,
+				'settings'        => null,
+				'expected_output' => null,
 			),
 
-			'returns value when fluid is `false`'        => array(
-				'font_size'                   => array(
+			'returns value when fluid config is empty`'  => array(
```

Comment:
It looks like this test is still testing when the fluid value is false. Was that the intention? If so, just a quick suggestion to restore the previous label:

```suggestion
			'returns value when fluid is `false`'  => array(
```